infrastructure/cdn-in-a-box/to_integration/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What about to_integration_test for this directory? or traffic_ops_integration_test?